File: Java/src/test/java/fr/katas/HelloWorldTest.java
package fr.katas;

import org.assertj.core.api.Assertions;
import org.junit.jupiter.api.Test;

class HelloWorldTest {

    @Test
    void Should_greet_the_world(){
        HelloWorld classUnderTest = new HelloWorld();
        Assertions.assertThat(classUnderTest.getGreeting()).isEqualTo("Hello World!");
    }

}

File: Kotlin/src/test/kotlin/fr/katas/HelloWorldTest.kt
package fr.katas

import org.assertj.core.api.Assertions
import kotlin.test.Test
import kotlin.test.assertEquals

class HelloWorldTest {
    @Test
    fun `Should greet the world`(){
        val greeter = App()

        Assertions.assertThat(greeter.greeting).isEqualTo("Hello, World!")
    }

}
